### PR TITLE
Update to 1.9 (1.9 Compability)

### PR DESCRIPTION
**Pull request by Brottweiler**
This is an update that will make the pack **work** for 1.9, but there are still updates and tweaks that should be done to make it "plast packy".

# Changes

- Updated language file
- Added Frosted Ice
- Updated Brewing stand
- Updated Creative inventory
- Updated Inventory and added missing potion effects
- Updated Stats icons
- Added missing icons
- Added missing widgets
- Added empty shield slot
- Added missing particles
- Updated version from `1` (pre-1.9) to `2` (1.9.x)

Related: https://github.com/Plastix/Plast-Pack/issues/54

### CODE REVIEW

**Comment by Brottweiler** [comment]
a2f56a4474bc76002a5bbb3d8bf6226f565360ed is an attempt to take the ice+destroy animation, the same as the default frosted ice works. It's just to match it though, might want something different like more transparency instead of destroy animation?

File: assets/minecraft/lang/en_OC.lang [hunk 1]
```diff
@@ -243,10 +316,9 @@ options.title=Options
 options.controls=§bControls
 options.video=§bVideo Settings
 options.language=§bLanguage
-options.stream=§bBroadcast Settings
 options.sounds=§bMusic & Sounds
-options.sounds.title=Music & Sound Options
-options.languageWarning=Language translations may not be 100% accurate
+options.sounds.title=§bMusic & Sound Options
+options.languageWarning=Language translations may not be 100%% accurate
```
Comment (Plastix) [inline assets/minecraft/lang/en_OC.lang]: Is there a reason for the second %?

File: assets/minecraft/lang/en_OC.lang [hunk 2]
```diff
@@ -413,6 +443,7 @@ key.right=§bStrafe Right
 key.jump=§bJump
 key.inventory=§bInventory
 key.drop=§bDrop Item
+key.swapHands=Swap Item In Hands
```
Comment (Plastix) [inline assets/minecraft/lang/en_OC.lang]: Add §b

File: assets/minecraft/lang/en_OC.lang (hunk 1 above)
Comment (Brottweiler) [inline assets/minecraft/lang/en_OC.lang]: @Plastix original string:
`options.languageWarning=Language translations may not be 100%% accurate`

Comment (Plastix) [inline assets/minecraft/lang/en_OC.lang]: Ok keep it.

**Comment by Plastix** [comment]
I think you need to remove the orange and blue bounds on the new potion effects in `inventory.png`

**Comment by Brottweiler** [comment]
@Plastix wops, fixed.

**Comment by Plastix** [comment]
Looks good. :+1:

**Comment by Brottweiler** [comment]
To the new potion effects I added these color codes just now:

```
effect.glowing=§aGlowing
effect.luck=§aLuck
effect.unluck=§cBad Luck
effect.levitation=§cLevitation
```
`§a` is green and good, `§c` is red and bad. I think Glowing (Spectral?) is good, and Levitation is bad.

**Comment by Brottweiler** [comment]
Ninja'd :scream:

**Comment by Plastix** [comment]
Sorry. Feel free to open a new PR with those language changes.

**Comment by unixfox** [comment]
Can you create a release (in this section : https://github.com/Plastix/Plast-Pack/releases) when the texture pack is done ?

**Comment by Brottweiler** [comment]
@unixfox Yes. At this time, I think a pre-release might be suitable, actually, since it works but not fully updated.

**Pull request merged**